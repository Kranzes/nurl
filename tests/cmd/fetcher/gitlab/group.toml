args = ["https://gitlab.gnome.org/World/Phosh/phosh", "v0.20.0"]
