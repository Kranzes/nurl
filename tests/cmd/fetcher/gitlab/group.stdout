fetchFromGitLab {
  domain = "gitlab.gnome.org";
  group = "World";
  owner = "Phosh";
  repo = "phosh";
  rev = "v0.20.0";
  hash = "sha256-KKoRAUqy9l/Y/R7iLqZe6sHcyHHvRRTj+WeVmyAm7+I=";
}